Raven.Voron/Voron/Trees/Page.cs

```diff
@@ -172,6 +172,19 @@ public ushort PageSize
             }
         }
 
+        public List<long> GetAllOverflowPages()
+        {
+            var results = new List<long>(NumberOfEntries);
```

Comment:
The point is you dont care at all if they are or they are not. At least for 3.0 and 3.5 we cannot know for sure without causing the page fault. So in essense what you do is request a prefetch anyways. If you end up accessing it, fine; if not it is wasted effort. But even if it is a single leaf of 4Kb you still gain because the prefetch did the work anyways.